app/services/apiEndpoints.js

```diff
@@ -1,4 +1,4 @@
-export const BASE_URL = API_BASE_URL;
+export const BASE_URL = API_BASE_URL; // eslint-disable-line no-undef
```

Comment:
Is there any chance we can import the variable instead of referencing it globally?

Reply:
Not sure, I don't want to change logic in the PR. Only upgrade of react, removing warnings, and that kind of tasks. Maybe can be reviewed later all the others improvements. Thanks